tests/docker-compose.yml

```
@@ -6,6 +6,10 @@ services:
     command: azurite -L -l /data --blobHost 0.0.0.0 --queueHost 0.0.0.0
     ports:
       - "10000"
+  motoserver:
+    image: motoserver/moto
+    ports:
+      - "5000:5000"
```

Looks like you are already logic for dynamic ports in `tests/remotes/s3.py`, so there shouldn't be any need to nail it down to 5000 on the host
```suggestion
      - "5000"
```